permit.md

```diff
@@ -1,6 +1,8 @@
 # Overview
 Tables: VPS_VESSEL; VPS_FISHERY_NER; BUS_OWN, VPS_OWNER
 
+*the BUS_OWN table is not the main view for business onwer information. THat is found here: Client.bus_own@garfo_nefsc. For more information on this system see Client.md
```

Comment:
* the BUS_OWN table is not the main view for business owner information. That is found here: Client.bus_own@garfo_nefsc. For more information on this system see [Client.md](Client.md)